src/pages/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -1,9 +1,224 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import EmailTemplate from '../../../../components/Dashboard/scenes/Emails/EmailTemplate';
+import EmailHistory from '../../../../components/Dashboard/scenes/Emails/EmailHistory';
+import { EMAILAPI_URL, EMAILAPI_SENDER } from '../../../../constants';
+import LoadingSmallSVG from '../../../../assets/svg/LoadingSmallSVG';
 
 const Emails: React.FC = () => {
+  const [selectedMentees, setSelectedMentees] = useState<string[]>([]);
+  const [subject, setSubject] = useState('');
+  const [body, setBody] = useState('');
+  const [view, setView] = useState('sent');
+  const [select, setSelect] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const handleFormSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const emailData = {
+      sender: `${EMAILAPI_SENDER}`,
+      recipients: selectedMentees,
+      subject,
+      body,
+    };
+    setIsLoading(true);
+    try {
+      const response = await fetch(`${EMAILAPI_URL}api/v1/send`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(emailData),
+      });
+
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.statusText}`);
+      }
+      setMessage('Success: Emails sent!');
+    } catch (error) {
+      setMessage(`Error : ${(error as Error).message}`);
+    } finally {
+      setIsLoading(false);
+      setTimeout(() => {
+        setIsLoading(false);
+      }, 1000);
+    }
+  };
+
+  useEffect(() => {
+    if (message.length > 0) {
+      const timer = setTimeout(() => {
+        setMessage('');
+      }, 1000);
+      return () => {
+        clearTimeout(timer);
+      };
+    }
+  }, [message]);
+
+  const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    switch (e.target.value) {
+      case 'allMentors':
+        setSelectedMentees(allMentorsEmails);
```

Comment:
This is supposed to be a string. Check the other lint errors as well

Reply:
still I could not implement this. I don't have how the API call should be to call and fetch specific emails from backend!

Reply:
for now, console.log the value. I will integrate it.